src/main/java/com/google/api/codegen/transformer/TestCaseTransformer.java

```diff
@@ -384,7 +384,11 @@ private InitCodeContext createResponseInitCodeContext(
       FieldModel field = config.getResourcesField();
       InitCodeNode initCodeNode;
       if (field.isRepeated()) {
-        initCodeNode = InitCodeNode.createSingletonList(config.getResourcesFieldName());
+        if (field.isMap()) {
```

Comment:
Add a brief comment that only the first `map` or `repeated` (list) field is used? This may not be obvious to folks who have not read the design doc.

Reply:
+1